doc/release-notes/release-notes-bucash1.2.0.0.md

```diff
@@ -0,0 +1,139 @@
+Release Notes for Bitcoin Unlimited Cash Edition 1.2.0.0
+=========================================================
+
+Bitcoin Unlimited Cash Edition version 1.2.0.0 is now available from:
+
+  <https://bitcoinunlimited.info/download>
+
+Please report bugs using the issue tracker at github:
+
+  <https://github.com/BitcoinUnlimited/BitcoinUnlimited/issues>
+
+This is a minor release version based of Bitcoin Unlimited compatible
+with the Bitcoin Cash specification you could find here:
+
+https://github.com/Bitcoin-UAHF/spec/blob/master/uahf-technical-spec.md
+
+
+Upgrading
+---------
+
+If you are running an older version, shut it down. Wait until it has completely
+shut down (which might take a few minutes for older versions), then run the
+installer (on Windows) or just copy over /Applications/Bitcoin-Qt (on Mac) or
+bitcoind/bitcoin-qt (on Linux).
+
+The first time you run this new version, your UTXO database will be converted
+to a new format. This step could take a variable amount of time that will depend
+on the performance of the hardware you are using.
```

Comment:
the utxo upgrade was done already in the prior release.

Reply:
might be useful to leave in but say "if you are upgrading from an older release..."

Reply:
fixed